apps/passport-server/src/apis/devconnect/devconnectPretixAPI.ts

```diff
@@ -97,12 +97,6 @@ export class DevconnectPretixAPI implements IDevconnectPretixAPI {
         }
       }
 
-      if (!result.ok) {
-        throw new Error(
-          `[PRETIX] Error fetching ${input}: ${result.status} ${result.statusText}`
-        );
-      }
-
```

Comment:
This exception was unnecessary, because all functions which call this function also check the result, and throw more specific errors. This change also makes `queuedFetch` behave more like `fetch`, which does not throw exceptions for non-OK HTTP responses.